mobile/src/main/java/org/openhab/habdroid/model/Widget.kt

```diff
@@ -178,6 +187,12 @@ data class Widget(
         return chartUrl.toString()
     }
 
```

Comment:
Is this actually called anywhere?

Reply:
it's unrelated to this PR? A quick search shows toChartUrl() called in ChartWidgetActivity.kt and WidgetAdapter.kt

Reply:
Sorry, the comment was meant to point to the newly added `updateButton` method.

Reply:
I've removed `updateButton`